src/mmw/js/src/core/utils.js

```diff
@@ -73,6 +73,54 @@ var utils = {
             });
 
         return md5(JSON.stringify(sortedCollection));
+    },
+
+    convertToMetric: function(value, fromUnit) {
```

Comment:
Consider modifying `convertToMetric` and `convertToImperial` to take in the desired output unit as a parameter. I think this could make it more clear when using these functions.